demo/captioned_image.cpp

```diff
@@ -42,6 +42,11 @@ CaptionedImage::CaptionedImage()
     this->forwardXMLAttribute("imageHeight", this->image, "height");
 
     this->forwardXMLAttribute("caption", this->label, "text");
+
+    this->addGestureRecognizer(new brls::TapGestureRecognizer([this](auto _){
```

Comment:
I don't really like usage of auto especially in that case where there is no clue as to what is the function argument here (no type, no name)

Reply:
I used it like that cause it is useless in that case, but ok, I'll replace it.

Reply:
It's useless I agree but I never used auto anywhere in the project and I don't intend to. There are a few places where auto is used from code by other contributors, I will probably go through all of them and remove them.